media-plugins/x42-plugins/x42-plugins-20200714.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Collection of LV2 plugins"
+HOMEPAGE="https://github.com/x42/x42-plugins"
+SRC_URI="http://gareus.org/misc/x42-plugins/${P}.tar.xz"
+KEYWORDS="~amd64"
+LICENSE="GPL-2"
+SLOT="0"
+RESTRICT="mirror"
+
+RDEPEND="dev-libs/glib
+	media-fonts/dejavu
+	media-libs/ftgl
+	media-libs/glu
+	media-libs/liblo
+	media-libs/libsamplerate
+	media-libs/libsndfile
+	media-libs/libltc
+	media-libs/lv2
+	media-libs/zita-convolver
+	sci-libs/fftw:3.0
+	virtual/jack
+	virtual/opengl
+	x11-libs/cairo[X]
+	x11-libs/pango
+"
+DEPEND="${RDEPEND}
+	sys-apps/help2man"
+
+src_compile() {
+	emake STRIP="#" FONTFILE="/usr/share/fonts/dejavu/DejaVuSans-Bold.ttf"
+}
+
+src_install() {
+	emake DESTDIR="${D}" PREFIX="/usr" install
```

Comment:
Probably `PREFIX="${EPREFIX}/usr"` is preferred here

Reply:
Ah, yeah, I expect I'll forget the EPREFIX stuff many times :x I'll update the ebuilds to include this.

Reply:
Just looked into this and tbh I'd like to not do the prefix stuff.
Changes to ebuilds are tricky because of interactions between src_configure, src_compile and src_install as well as the EPREFIX/ED/EROOT variables. I have no reason to use it myself and we currently have no way to automatically test if it's correct. Also the packages in this overlay are pretty much targeted at a niche (Linux audio) within a niche (Gentoo). If someone wants Linux audio packages they should just install it in their system's OS.
We already barely have time to maintain what we have and to be frank I don't have the time to figure out stuff that at least to me is unclear and brings very limited value. Because of this I'd prefer to not do anything with regards to prefix.

Reply:
My idea was to just dump `${EPREFIX}` in front of any paths passed to non-builtins and call it a day (and maybe use ED for any direct image accesses), since a bunch of upstream ebuilds do that too. If you feel uncomfortable doing that then it's fine not to do it; I just figured we might as well attempt to support such use cases even if we don't test it.